rows/operations.py

```diff
@@ -28,7 +28,7 @@ def join(keys, tables):
 
     # Make new (merged) Table fields
     fields = OrderedDict()
-    map(lambda table: fields.update(table.fields), tables)
+    list([fields.update(table.fields) for table in tables])
```

Comment:
This line may be refactored as a regular `for` loop (I don't know why I didn't use it), like this:

```python
for table in tables:
    fields.update(table.fields)
```